src/packageurl/contrib/url2purl.py

```diff
@@ -276,3 +294,100 @@ def build_sourceforge_purl(uri):
 @purl_router.route(cargo_pattern)
 def build_cargo_purl(uri):
     return purl_from_pattern('cargo', cargo_pattern, uri)
+
+
+@purl_router.route('https?://github.com/.*')
+def build_github_purl(uri):
+    """
```

Comment:
What does this function do and return? Tell that in the doctrsing :)

Reply:
@TG1999 The docstring should be more descriptive (see @pombredanne comment above)

Reply:
Yeah sure, sorry for this mistake